### ISSUE TITLE
toolTip:tooltipIsOpen

### ISSUE THREAD

**Issue by OlegMal88** [valor-software/ngx-bootstrap#883]
Can't get the value of tooltipIsOpen flag. Add please eventEmitter with new state of toolTip.

**Comment by matthewdaniel** [comment]
i think it is implemented in such a way that you can progromatically set the isOpen value from the outside, it isn't designed to be an event. 

That being said, if that is the purpose then it isn't working that way either

**Comment by adrianfaciu** [comment]
It looks like isOpen was not implemented (yet). I'll create a pull request once the feat-rc5 branch is merged into main.

**Comment by hwupu** [comment]
How was it? I can't get it to work as well.

**Comment by adrianfaciu** [comment]
Fixed with #939.
